.github/workflows/integration-tests-against-emulator-with-regular-session.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v5
         with:
-          python-version: 3.8
+          python-version: 3.12
```

Comment:
this is because in followup PR we will be removing 3.8 runtime support
https://github.com/googleapis/python-spanner/pull/1395